pkg/operator/ceph/csi/csi.go

```diff
@@ -48,7 +48,16 @@ func (r *ReconcileCSI) validateAndConfigureDrivers(ownerInfo *k8sutil.OwnerInfo)
 	}
 
 	// Check whether RBD or CephFS needs to be disabled
-	return r.stopDrivers()
+	err = r.stopDrivers()
+	if err != nil {
+		return errors.Wrap(err, "failed to stop Drivers")
+	}
+
+	err = r.deleteCSIDriverObject()
```

Comment:
We are stopping individual drivers ( in`stopDrivers()`) based on whether its disabled or not,    but then deleting all the drivers.  Shouldn't we also delete individual drivers (in `deleteCSIDriverObject` ), like we do in `stopDrivers` ?